setup.py

```diff
@@ -24,18 +24,18 @@ def get_package_data(package):
 
 
 setup(
-    name='django-admin-select2',
-    version='1.0.1',
-    description='Enable select2 for Django admin select inputs',
+    name='django3_admin_select2',
+    version='1.0.0',
```

Comment:
Lets start with version 0.1.0, when we are sure it's all working, we bump it to 1.0